scripts/check_elf_alignment.sh

```
@@ -0,0 +1,113 @@
+case ${1} in
+  --help | -h | -\?)
+    usage
+    exit
+    ;;
+
+  *)
+    dir="${1}"
+    ;;
+esac
+
+if ! [ -f "${dir}" -o -d "${dir}" ]; then
```

Shellcheck (0.11.0)

[warning] Prefer [ p ] || [ q ] as [ p -o q ] is not well defined.

(SC2166)